private_repo/ansible/inventory/dev/KnowledgePlatform/secrets.yml

```diff
@@ -48,4 +48,16 @@ lp_vault_graph_passport_key: "long-secret-to-calm-entropy-gods"
 
 # update if seperate object storage is used
 cloud_private_storage_secret: "{{ cloud_public_storage_secret }}"
-cloud_artifact_storage_secret: "{{ cloud_public_storage_secret }}"
\ No newline at end of file
+cloud_artifact_storage_secret: "{{ cloud_public_storage_secret }}"
+
+### Lern BB
```

Comment:
Add relevant comments to all the vars. Just reuse same comments from other inventory.

Reply:
Updated, pls check